README.md

```diff
@@ -104,7 +151,6 @@ Here are some things to try.
 
 Once installed as described above, type
 
-    $ source venv/3.10/bin/activate
```

Comment:
You can add this back and just make a comment to activate the venv if it's not already